modules/javafx.graphics/src/main/java/com/sun/javafx/tk/Toolkit.java

```diff
@@ -96,6 +94,9 @@ public abstract class Toolkit {
     private static Toolkit TOOLKIT;
     private static Thread fxUserThread = null;
 
+    // Used as the (dummy) value for the various listener maps
+    private static final Object dummyObj = new Object();
```

Comment:
minor: maybe use all-uppercase name for this static final object?

we could also be using Boolean.TRUE (or null), since there is a comment down below explaining the purpose of these maps.

curiously, there is no WeakHashSet.

Reply:
There is no need for `WeakHashSet`, because it would be equivalent to `Collections.newSetFromMap(new HashMap<>())`.
`HashSet` already uses `HashMap` in its implementation.

Reply:
wow, I did not they _cut corners_ there.  who is going to look after little peo^H^H^H bytes?